tasks/bundle_analysis_processor.py

```diff
@@ -156,6 +156,8 @@ def process_impl_within_lock(
         finally:
             if result.bundle_report:
                 result.bundle_report.cleanup()
```

Comment:
Close the loaded commit's parent's bundle analysis report, this should fix a lot of hanging SQLite threads that was seen in Sentry logs.